### PR TITLE
Saito ERC20

### PR DESCRIPTION
**Pull request by clayrab**
This PR is being closed due to inactivity. If you wish to continue, please have us reopen the PR before sending your payment, or just create a new one.
Do NOT send payments for closed PR, as the fee may by lost!
  (48 vs. 48 hrs)

### CODE REVIEW

**Comment by clayrab** [comment]
We already sent payment to this...

**Comment by clayrab** [comment]
https://explorer.binance.org/tx/1A6DD64F37474B7551588CA8B44CBA8CE7BBCD9BA5071352C43D21F53A4C3AA9

File: blockchains/ethereum/assets/0xFa14Fa6958401314851A17d6C5360cA29f74B57B/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "SAITO Token",
+    "website": "https://saito.io",
+    "description": "ERC-20 SAITO",
+    "type": "ERC-20",
```
Comment (vikmeup) [inline blockchains/ethereum/assets/0xFa14Fa6958401314851A17d6C5360cA29f74B57B/info.json:5]:
```suggestion
    "type": "ERC20",
```

File: blockchains/ethereum/assets/0xFa14Fa6958401314851A17d6C5360cA29f74B57B/info.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "SAITO Token",
+    "website": "https://saito.io",
+    "description": "ERC-20 SAITO",
```
Comment (vikmeup) [inline blockchains/ethereum/assets/0xFa14Fa6958401314851A17d6C5360cA29f74B57B/info.json:4]: can you provide proper description?

Comment (clayrab) [reply blockchains/ethereum/assets/0xFa14Fa6958401314851A17d6C5360cA29f74B57B/info.json:4]: yes, one moment

**Comment by clayrab** [comment]
Looks okay?

**Pull request merged**